src/System.Runtime/tests/System/Runtime/Serialization/OnDeserializedAttributeTests.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.Serialization.Tests
+{
+    public class OnDeserializedAttributeTests
+    {
+        [Fact]
+        public void Ctor_Default()
```

Comment:
whts the purpose oh this test ?

Reply:
I'm gonna remove these tests and they don't pull any weight - I wanted to make sure they do not throw